CONTRIBUTORS.md

```diff
@@ -343,9 +342,7 @@
 
 -[@hashcodepk](https://github.com/Hashcodepk)
 
--[@uniqueayo1988](https://github.com/uniqueayo1988)
--[@kumarnitinsingh](https://github.com/kumarnitinsingh/)
--[@krishnanath](https://github.com/krishnanath)
+-[@uniqueayo1988](https://github.com/uniqueayo1988) -[@kumarnitinsingh](https://github.com/kumarnitinsingh/) -[@krishnanath](https://github.com/krishnanath)
```

Comment:
Bunching other students up like this is not ideal. 
Any chance you could correct this?